elliott/elliottlib/cli/attach_cve_flaws_cli.py

```diff
@@ -150,7 +151,23 @@ def get_flaws(flaw_bug_tracker: BugTracker, tracker_bugs: Iterable[Bug], brew_ap
 def _update_advisory(runtime, advisory, flaw_bugs, bug_tracker, noop):
     advisory_id = advisory.errata_id
     errata_config = runtime.get_errata_config()
-    cve_boilerplate = errata_config['boilerplates']['cve']
+
+    # Get the advisory name, eg. image|rpm|metadata|microshift
+    releases_config = runtime.get_releases_config()
+    advisories = releases_config[runtime.assembly]['assembly']['group']['advisories']
```

Comment:
i think there should be a better method which returns assembly config as Model then you can just do
`assembly_config.group.advisories`

Reply:
have a suggestion, instead of doing this here, it would be better if you passed `advisory_by_kind: dict` to this func. this advisory_by_kind can be prepared initially (see line 60-70)

Reply:
well if the goal is fail fast, I've made the update here: https://github.com/openshift-eng/art-tools/pull/1573/commits/e938741b82d243e6468f17776d8990c9a754d43a

I checked other places in the code base and this is how we were retrieving the value
```
releases_config["releases"][runtime.assembly]['assembly']['group']['advisories']
```
and I wanted it to fail fast too. if we change it to a model, we have to write additional lines to check if its a missing Model or not.